app/services/operations.rb

```diff
@@ -0,0 +1,2 @@
+module Operations
+end
```

Comment:
Not sure if having this empty module here is worth ... I'd probably :scissors:

Reply:
what do you think @Morantron about this :scissors:? It causes problems with Rails autoloading without this top-level module?